index.html

```diff
@@ -15,9 +15,25 @@
 
 <body>
     <div class="home-page">
-        <header class="navbar">
+        <!--Toolbar Begins-->
+        <div class="mobile-menu">
+            <div class="mobile-menu-close-toggle-container">
+                <a id="mobile-menu-close-toggle" onclick="toggleClasses()">
+                    <img src="./images/icon_cancel.svg" alt="" class="mobile-menu-close-toggle-image">
```

Comment:
- [ ] Please edit the close button color to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/188927215-ad7a1164-24aa-4dcb-b8a0-57eb62eb1692.png) | ![image](https://user-images.githubusercontent.com/24703171/188927352-42352ac2-7f33-4131-b6a0-e20e8f9684b8.png) |